src/app/game-state/inventory.service.ts

```diff
@@ -998,16 +1007,22 @@ export class InventoryService {
   }
 
   // a special use function for generated potions
-  usePotion(potion: Potion){
-    this.lifetimePotionsUsed++;
-    this.characterService.characterState.attributes[potion.attribute].value += potion.increase;
+  usePotion(potion: Potion, count?: number){
+    if(!count){
+      count = 1;
+    }
+    this.lifetimePotionsUsed += count;
+    this.characterService.characterState.attributes[potion.attribute].value += potion.increase * count;
   }
 
   // a special use function for generated pills
-  usePill(pill: Pill){
-    this.lifetimePillsUsed++
+  usePill(pill: Pill, count?: number){
+    if(!count){
+      count = 1;
+    }
```

Comment:
```suggestion
  usePill(pill: Pill, count = 1){
```

Reply:
I could default it... or I could ensure 0 and undefined are both 1 so I don't have to do 0 later.